packages/gatsby-source-filesystem/index.d.ts

```diff
@@ -103,7 +103,7 @@ export interface FileSystemConfig {
 }
 
 /**
- * @see https://www.gatsbyjs.org/packages/gatsby-source-filesystem/?=filesy#options
+ * @see https://www.gatsbyjs.com/packages/gatsby-source-filesystem/?=filesy#options
```

Comment:
Should these be updated to `/plugins` instead of `/packages`?

Reply:
I can! However the redirect for `/packages` -> `/plugins` is in place so I didn't bother.